src/main.cpp

```diff
@@ -7,6 +7,8 @@
 #include <BLEUtils.h>
 #include <BLEServer.h>
 
+#include <esp_gap_ble_api.h>
```

Comment:
Any reason for this include? I am able to compile fine without it.

Reply:
I introduced esp_gap_ble_api.h while experimenting with some related functionalities during testing. It seems I missed removing it during code cleanup. Now, it appears that this inclusion is no longer necessary. Thank you for pointing it out!